default.nix

```diff
@@ -10,7 +11,9 @@ let
   inherit (specs.this) deps;
 
   # Import deploy scripts from dss-deploy
-  dss-deploy = import deps.dss-deploy.src' { inherit pkgs; };
+  dss-deploy' = if (dss-deploy != null)
```

Comment:
there should be a `lib.isNull`, so you could do `if isNull dss-deploy then`.

Reply:
You're right! It's even in `builtins` :)